view/script/patient.js

```diff
@@ -38,7 +39,9 @@
         }
     };
 
+    var allDatesConfig = jQuery.extend(true, {}, window.dates);
```

Comment:
what does this do?
is this adding an empy object and all the dates to a primitive boolean value? :confused: 

reference: https://api.jquery.com/jquery.extend/